.github/workflows/build.yml

```diff
@@ -872,6 +872,7 @@ jobs:
       - name: Execute tests within container
         run: |
           docker run --name=front-test --net=host -v $PWD/output:/app/tests/unit \
+            -e PROXY_EDITOAST='/api' -e PROXY_GATEWAY='' \
```

Comment:
Since these are the defaults, maybe we can just leave them unspecified?

Reply:
import-meta-env expects them to be defined also in the container here